ethereum/llama/aave_treasury/aave_daily_atoken_balances.sql

```diff
@@ -0,0 +1,15 @@
+CREATE SCHEMA IF NOT EXISTS llama;
+
+CREATE TABLE IF NOT EXISTS llama.aave_daily_atoken_balances (   
+	day timestamptz,
+	token_address bytea,
+	daily_change numeric,
+	starting_balance numeric,
+	interest_rate_apr numeric,
+	int_earned numeric,
+	total_bal numeric,
```

Comment:
Any reason for shortening `total_bal`, could be `total_balance` otherwise?

Reply:
updated